shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/event/SerializableListenerEventFactory.java

```diff
@@ -30,6 +30,14 @@
 import org.apache.hadoop.hive.metastore.events.ListenerEvent;
 import org.apache.hive.common.util.HiveVersionInfo;
 
+import com.hotels.shunting.yard.common.event.apiary.SerializableApiaryAddPartitionEvent;
```

Comment:
I think we should because these events are specific to Apiary. If tomorrow, we have another service Xyz, sending us events in some other format, then we can name those events in a similar manner "SerializableXyzAddPartitionEvent".